oaknational/Oak-Web-Application
# Feat/upcoming webinar

## Description

- Adds `UpcomingWebinarListItem` component
- Adds bg image to `WebinarWall` component
- Allows passing of title and text as props to `WebinarWall` to allow it to be used on upcoming webinars too

## Issue(s)

Progresses #879 
Prgoresses #878 

## How to test

1. Go to {storybok deploy}
2. Check "Blogs > Blog List > Upcoming Webinar List Item"
3. Check "Blogs > WebinarWall"

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/200275419-62b10b10-f812-45fc-8f90-ce3d71f62834.png)


![image](https://user-images.githubusercontent.com/12934669/200275367-7cc0462d-c8fd-49f1-9680-27588a175c69.png)



## Checklist
